### PR TITLE
Fix multiplication for Stream object

### PR DESCRIPTION
**Pull request by carlogrisetti**
This is the edit done in issue #284 implemented in a PR

### CODE REVIEW

**Comment by carlogrisetti** [comment]
@rhamnett can you take a look at this, since I'm not that into Streams? :)

File: tensortrade/env/default/observers.py [hunk 1]
```diff
@@ -43,7 +44,14 @@ def _create_wallet_source(wallet: 'Wallet', include_worth: bool = True) -> 'List
...
-            worth = (price * total_balance).rename("worth")
+
+            if price.value is None:
+                price.value = price.forward()
+
+            if total_balance.value is None:
+                total_balance.value = total_balance.forward()
+
+            worth = mul(price, total_balance).rename('worth')
             streams += [worth]
```
Comment (rhamnett) [inline tensortrade/env/default/observers.py:55]: I feel like price should never be none and earlier in the process there should be a fill forward done on the original dataset so raising an error might be more preferable. @mwbrulhardt  could you please chime in?

Comment (rhamnett) [reply tensortrade/env/default/observers.py:55]: Also if we forward() here will that end up being out of sync?

Comment (mwbrulhardt) [reply tensortrade/env/default/observers.py:55]: Yea a price should never be`None` coming into this context. I believe I have unit tests that check this function.

Comment (rhamnett) [reply tensortrade/env/default/observers.py:55]: The user in the issue uses a custom downloader for data which probably contains errors or no ffil() etc...and so therefore I suggest we go back to him for a cleaner test case.

**Comment by rhamnett** [comment]
The user in the issue uses a custom downloader for data which probably contains errors or no ffil() etc...and so therefore I suggest we go back to him for a cleaner test case.

**Comment by carlogrisetti** [comment]
Should we start closing this only with the `worth = mul(price, total_balance).rename('worth')` edit, and leaving the "is None" part to a specific issue should that need arise?

File: tensortrade/env/default/observers.py [hunk 2]
```diff
@@ -44,13 +44,6 @@ def _create_wallet_source(wallet: 'Wallet', include_worth: bool = True) -> 'List
...
         if include_worth:
             price = Stream.select(wallet.exchange.streams(), lambda node: node.name.endswith(symbol))
-
-            if price.value is None:
-                price.value = price.forward()
-
-            if total_balance.value is None:
-                total_balance.value = total_balance.forward()
-
             worth = mul(price, total_balance).rename('worth')
```
Comment (rhamnett) [inline tensortrade/env/default/observers.py:47]: This doesn't work. I think syntax is
```python
worth = price.mul(total_balance).rename('worth')
```

**Comment by rhamnett** [comment]
completed in #305

**Pull request closed without merging**